README.md

```diff
@@ -1,3 +1,59 @@
-Add some basic description of the project here (JavaScript virtual pet project).
+Virtual Pet
```

Comment:
The formatting and content _really_ needs looking at here.

- The table of contents is not required at all so could be removed
- There is zero use of markdown, links and line spacing here so its not at all easy to read quickly
- How do you tell where one section ends and another begins?
- You dont need to include all the info about things like the `.gitignore` file, its not specific to this project

I wouldnt normally do this, but as an example Ill show you what this could look like:
```
# Virtual Pet

A JavaScript educational project, completed as part of the Software Development FastTrack course at Command Shift (Manchester Codes).

(You should add some content about what this project actually does here!)

Acknowledgement to former student Jen Openshaw & Rita Łyczywek for this very useful 'Virtual Pet' README template.

## Features
List of current features:

- Pets can be given names
- Pets can grow older
- The hunger and fitness levels of pets can be tracked
- Pets get hungry and need to be fed
- Pets get hungrier as they get older
- Pets lose fitness as they get older
- It is possible to check status of pet
- Pet dies if it gets too old, too hungry or too unfit
- Pet can adopt children

## Installation/usage/testing instructions
(note use of ### subheadings here)
### Installation
Describe how to install the app. Can you provide a step by step guide?

### Testing
How is the app tested?

### Usage
How is the app used?

## Technologies and Languages
Add a list here

## To-do list for future development:
Create a GUI, as the application can only be ran as a test in Node or Jest

## Contact
- LinkedIn: [anthonymmoran](linkedin.com/in/anthonymmoran)
- Twitter: [@TonyMCodes](twitter.com/TonyMCodes)
- [Command Shift (Manchester Codes)](https://www.commandshift.co/)

```

I would highly recommend looking into how markdown works, its really important for writing good readmes and documentation. [Markdown live preview](https://markdownlivepreview.com/) is useful for playing with what the file will look like without having to do multiple commits to the repo to see the results, have a look through this [guide to markdown](https://www.markdownguide.org/cheat-sheet/) too

Reply:
Thank you for this feedback Stu. I've never heard of markdown before and so will look into it! I don't know how to add images to a readme either, so will also investigate that also.

Reply:
Its very easy as it happens, the format is 
```
![some alt text here ](http://url/to/img.png)
```
if have to upload the image to your github account for it to appear, usually you would put this in an images folder at the root of the project since it isnt part of the ocde

Reply:
OK, great - I'll give it a go now, cheers!